terraso_backend/apps/core/models/users.py

```diff
@@ -114,6 +115,36 @@ def soft_delete_policy_action(self, **kwargs):
     def is_group_manager(self, group_id):
         return self.memberships.managers_only().filter(group__pk=group_id).exists()
 
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}".strip()
+
+    def name_and_email(self):
+        return f"'{self.full_name()}' <{self.email}>"
+
+    def notifications_enabled(self):
+        preferences = self.preferences.filter(key="notifications")
+        if len(preferences) != 1:
+            return False
+
+        try:
+            return preferences[0].value.lower() == "true"
+        except AttributeError:
+            return False
+
+    def language(self):
+        preferences = self.preferences.filter(key="language")
+        if len(preferences) != 1:
+            return settings.LANGUAGE_CODE
```

Comment:
Suggestion: `LANGUAGE_CODE` seems to be the default language code, I think `LANGUAGE_CODE_DEAFULT` or `DEFAULT_LANGUAGE_CODE` will make this easier to understand

Reply:
I agree! However, `LANGUAGE_CODE` is a special Django configuration value: https://docs.djangoproject.com/en/4.1/ref/settings/. We could duplicate that i.e.
`DEFAULT_LANGUAGE_CODE = LANGUAGE_CODE` in settings.py if you wanted to improve readibility.

Reply:
I think reassing it to `DEFAULT_LANGUAGE_CODE = LANGUAGE_CODE` makes sense to me, but just a suggestion